challenge.js

```diff
@@ -57,5 +60,10 @@
  */
 
 // ANSWER
+var element = document.createElement("div");
+element.id = 'testqq';
+var el = document.getElementById('testqq');
```

Comment:
`document.getElementById` can only retrieve elements that are on the DOM. `document.createElement` creates a new HTML element object, but it is not attached to the DOM until you explicitly attach it somewhere (makes sense, right? If the browser attached it without being told where to do so, it would likely put it in the wrong place).

The desired answer was a link to the MDN page for `Node.appendChild`. 1/4